keanu-python/keanu/vertex/base.py

```diff
@@ -76,6 +77,12 @@ def get_label(self) -> Optional[str]:
         label = self.unwrap().getLabel()
         return None if label is None else label.getQualifiedName()
 
+    def get_parents(self) -> Iterator['Vertex']:
```

Comment:
nit: the applied have raised naming concerns for calling iterators `get_`. Let's either fix that as part of this PR or make sure we remember to fix these later

Reply:
I haven't heard them say that - let's discuss. In any case we should fix that outside of this PR.

Reply:
Let's address that outside of this PR since it affects many methods in `net` and `vertex.base`